### PR TITLE
pcb2gcode: fix build on darwin

### PR DESCRIPTION
**Pull request by cransom**
This version bumps gerbv as well.

###### Description of changes

Version bumps and changes the source for a dependency, gerbv, to sourceforge which seems to be the official download spot.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/science/electronics/gerbv/default.nix [hunk 1]
```diff
@@ -21,15 +20,19 @@ stdenv.mkDerivation {
   ];
 
   nativeBuildInputs = [ autoreconfHook pkg-config automake autoconf ];
-  buildInputs = [ gettext libtool cairo gtk2 ];
+  buildInputs = [ gettext libtool cairo gtk2-x11.dev ];
```
Comment (wegank) [inline pkgs/applications/science/electronics/gerbv/default.nix:23]: `.dev` might not be necessary?

File: pkgs/applications/science/electronics/gerbv/default.nix [hunk 2]
```diff
@@ -21,15 +20,19 @@ stdenv.mkDerivation {
...
 
   configureFlags = ["--disable-update-desktop-database"];
 
+  CFLAGS = [
+    "-Wno-format-security"
+  ];
```
Comment (wegank) [inline pkgs/applications/science/electronics/gerbv/default.nix:29]: Does the following work for you?

```suggestion
  env.NIX_CFLAGS_COMPILE = "-Wno-format-security";
```

Comment (cransom) [reply pkgs/applications/science/electronics/gerbv/default.nix]: Yep, works for me. Out of curiosity, I did some digging and I see [discussion on about phasing out NIX_CFLAGS_COMPILE](https://github.com/NixOS/nixpkgs/issues/79303). Is there a source on where I can learn about the nuance between them?

**Comment by cransom** [comment]
@wegank Updated. Thanks for looking!

File: pkgs/applications/science/electronics/gerbv/default.nix (hunk 2 above)
Comment (wegank) [reply pkgs/applications/science/electronics/gerbv/default.nix]: I consider `env.NIX_CFLAGS_COMPILE` to be more uniform. Other environmental variables could of course work here or there.

**Pull request merged**